app/components/preprint-form-authors.js

```diff
@@ -10,26 +10,158 @@ export default CpPanelBodyComponent.extend(PreprintFormFieldMixin, {
-        },
-        updateContributors() {
             this.sendAction(
                 'editContributors',
                 this.get('contributors'),
-                this.get('permissionChanges'),
+                {},
                 this.get('bibliographicChanges')
             );
+            this.set('bibliographicChanges', {});
+            this.toggleProperty('bibliographicToggle');
+            this.toggleProperty('removalToggle');
+        },
+        // There are 3 view states on left side of Authors panel.  This switches to add unregistered contrib view.
+        unregisteredView() {
+            this.set('addState', 'unregisteredView');
+        },
+        // There are 3 view states on left side of Authors panel.  This switches to searching contributor results view.
+        searchView() {
+            this.set('addState', 'searchView');
+            this.set('fullName', '');
+            this.set('email', '');
+        },
+        emptyView() {
+            this.set('addState', 'emptyView');
+            this.set('searchQuery', '');
+            Ember.$('.searchQuery')[0].value = '';
+        }
+    },
+    // Temporary popover content for authors page. Will need to find alternative to jquery selectors.
+    didInsertElement: function() {
+        Ember.$('#permissions-popover').popover({
+            content: '<dl>' +
+                '<dt>Read</dt>' +
+                    '<dd><ul><li>View preprint</li></ul></dd>' +
+                '<dt>Read + Write</dt>' +
+                    '<dd><ul><li>Read privileges</li> ' +
+                        '<li>Add and configure preprint</li> ' +
+                        '<li>Add and edit content</li></ul></dd>' +
+                '<dt>Administrator</dt><dd><ul>' +
+                    '<li>Read and write privileges</li>' +
+                    '<li>Manage authors</li>' +
+                    '<li>Public-private settings</li></ul></dd>' +
+                '</dl>'
+        });
+        Ember.$('#bibliographic-popover').popover({
+            content: 'Only checked authors will be included in preprint citations. ' +
+            'Authors not in the citation can read and modify the preprint as normal.'
+        });
+        Ember.$('#author-popover').popover({
+            content: 'Preprints must have at least one registered administrator and one author showing in the citation at all times.  ' +
+            'A registered administrator is a user who has both confirmed their account and has administrator privileges.'
+        });
+    },
+    /**
+    * If user removes their own admin permissions, many things on the page must become
+    * disabled.  Changing the stillAdmin flag to false will remove many of the options
+    * on the page.
+    */
+    removedSelfAsAdmin(contributor, permission) {
+        if (this.get('currentUser').id === contributor.id.split('-')[1] && permission !== 'ADMIN') {
+            this.set('stillAdmin', false);
```

Comment:
So what I mean by stillAdmin...when the page is loaded the user might be an admin. But if I change the permissions, it might no longer be an admin. The original isAdmin isn't updated, that was something I passed in as an argument.